hub/management/commands/import_gbgw_events_24.py

```diff
@@ -0,0 +1,42 @@
+from django.conf import settings
+
+from hub.models import DataSet
+
+from .base_importers import BaseConstituencyCountImportCommand, MultipleAreaTypesMixin
+
+
+class Command(MultipleAreaTypesMixin, BaseConstituencyCountImportCommand):
+    help = "Import data about number of GBGW events in 2024 per constituency"
+    message = "Importing 2024 GBGW events"
+    uses_gss = False
+
+    data_file = settings.BASE_DIR / "data" / "gbgw_events_24_processed.csv"
+
+    area_types = ["WMC", "WMC23", "STC", "DIS"]
```

Comment:
When I generate `gbgw_events_24_processed.csv`, the `WMC` column is empty. I assume this is intentional, because geocoding a lonlat to pre-2024 constituencies with MapIt is a pain.

So I think you need to remove `"WMC"` from the `area_types` here, so that the base importer doesn’t try to create data for (pre-2024) WMCs.

If you leave `"WMC"` in this list, without a column of WMCs in `gbgw_events_24_processed.csv`, you’ll get this error:

```
  File "/app/hub/management/commands/base_importers.py", line 81, in get_cons_col
    return self.cons_col_map[self.area_type]
KeyError: 'WMC'
```